quantlib/termstructures/yields/rate_helpers.pyx

```diff
@@ -71,12 +72,18 @@ cdef class RelativeDateRateHelper:
 cdef class DepositRateHelper(RateHelper):
     """Rate helper for bootstrapping over deposit rates."""
 
-    def __init__(self, Quote quote, Period tenor=None, Natural fixing_days=0,
-        Calendar calendar=None, int convention=ModifiedFollowing,
-        end_of_month=True, DayCounter deposit_day_counter=None,
+    def __init__(self, quote, Period tenor = None, Natural fixing_days=2,
+        Calendar calendar = None, int convention=ModifiedFollowing,
+        bool end_of_month=True, DayCounter deposit_day_counter=None,
         IborIndex index=None):
-        cdef Handle[_qt.Quote] rate_handle = Handle[_qt.Quote](deref(quote._thisptr))
-        
+        cdef Handle[_qt.Quote] rate_handle
+
+        if isinstance(quote, float):
+            rate_handle = Handle[_qt.Quote](shared_ptr[_qt.Quote](
+                new _qt.SimpleQuote(<Rate>quote)))
+        elif isinstance(quote, SimpleQuote):
+            rate_handle = Handle[_qt.Quote](deref((<SimpleQuote>quote)._thisptr))
+
```

Comment:
you might want to raise a `ValueError` if the `quote` is neither a float or a SimpleQuote.